src/sidebar/panel.js

```diff
@@ -6,15 +10,73 @@ var quill = new Quill('#editor', {
   }
 });
 
-chrome.storage.local.get("notes", function(data) {
-  quill.setContents(data["notes"]);
+function handleLocalContent(data) {
+  if(!data.hasOwnProperty("notes")) {
+    quill.setContents({
+      "ops": [
+        {"attributes": {"size": "large", "bold":true}, "insert": "Welcome!"},
+        {"insert": "\n\n"},
+        {"attributes": {"size": "large"},
+         "insert": "This is a simple one-page notepad built in to Firefox that helps you get the most out of the web."},{"insert": "\n\n"},
+        {"attributes": {"size": "large"}, "insert": "You can: "},
+        {"insert": "\n\n"},
+        {"attributes": {"size": "large"}, "insert": "Format your notes"},
+        {"attributes": {"list": "ordered"}, "insert": "\n"},
+        {"attributes": {"size": "large"}, "insert": "Sync notes securely to your Firefox Account"},
+        {"attributes": {"list": "ordered"}, "insert": "\n"},
+        {"attributes": {"size": "large"},
+         "insert": "Sync them to our Android app: http://mzl.la/notes"},
+        {"attributes": {"list": "ordered"}, "insert": "\n"}]});
+  } else {
+    quill.setContents(data["notes"]);
+  }
+}
+
+browser.storage.local.get(["bearer", "keys", "notes"], function(data) {
+  // If we have a bearer, we try to save the content.
```

Comment:
for next steps, do we need to check if the user has the 'default' placeholder text, e.g `Welcome!` and overwrite it after a login? I guess if it is not default text then we merge the records on login?